src/main/java/com/minecolonies/rankup/util/RankingUtils.java

```diff
@@ -78,42 +67,42 @@ public static void balanceCheck(final Player player, final Rankup plugin)
 
     public static void timeDown(Player player, Rankup plugin)
     {
-        final GroupsConfig groupsConfig = (GroupsConfig) plugin.getAllConfigs().get(GroupsConfig.class);
-        final AccountConfigData playerData = (AccountConfigData) plugin.getAllConfigs().get(AccountConfigData.class);
-
-        final List<String> playerGroups = CoreModule.perms.getPlayerGroupIds(player);
+        final GroupsConfig groupsConfig = plugin.configUtils.getGroupsConfig(player);
+        final List<String> playerGroups = plugin.perms.getPlayerGroupIds(player);
 
         //Check if player is in disabled group.
-        for (final Subject subject : CoreModule.perms.getDisabledGroups())
+        for (final String subject : plugin.perms.getDisabledGroups())
         {
-            if (playerGroups.contains(subject.getIdentifier()))
+            if (playerGroups.contains(subject))
             {
                 return; //IF the player is withing a disabled group, stop here.
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)